cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -98,7 +98,7 @@ A web application should make use of cookies for session ID exchange management.
 
 ### Transport Layer Security
 
-In order to protect the session ID exchange from active eavesdropping and passive disclosure in the network traffic, it is essential to use an encrypted HTTPS (TLS) connection for the entire web session, not only for the authentication process where the user credentials are exchanged.
+In order to protect the session ID exchange from active eavesdropping and passive disclosure in the network traffic, it is essential to use an encrypted HTTPS (TLS) connection for the entire web session, not only for the authentication process where the user credentials are exchanged. This can be assured with HTTP Strict Transport Security (HSTS).
```

Comment:
This is not entirely true because HSTS is a browser header. If you are using client that is not supporting HSTS adding this harder will not add any value. Moreover, the word 'this' can be misinterpreted. 

I suggest to either drop this change or make clear here what HSTS header is doing etc.

Reply:
Ok actually I think that we should just link to the 
https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Strict_Transport_Security_Cheat_Sheet.html

Reply:
Agree on both points